pkg/virt-controller/watch/vm.go

```diff
@@ -772,6 +788,17 @@ func (c *VMController) setupVMIFromVM(vm *virtv1.VirtualMachine) *virtv1.Virtual
 	vmi.ObjectMeta.Namespace = vm.ObjectMeta.Namespace
 	vmi.Spec = vm.Spec.Template.Spec
 
+	if len(vm.Status.StateChangeRequests) != 0 {
+		stateChange := vm.Status.StateChangeRequests[0]
+		if stateChange.Action == virtv1.StartRequest {
+			paused, hasPaused := stateChange.Data[virtv1.StartRequestDataPausedKey]
+			if hasPaused && paused == virtv1.StartRequestDataPausedTrue {
+				strategy := virtv1.StartStrategyPaused
+				vmi.Spec.StartStrategy = &strategy
+			}
+		}
+	}
```

Comment:
can we put this in a util function. something like

hasStartPausedRequest(vm) bool

Reply:
Done